api/controllers/roomController.ts

```diff
@@ -0,0 +1,51 @@
+import { Request, Response } from "express";
+import roomService from "../services/roomService";
+
+class roomController {
+  static getRoomList = async (req: Request, res: Response) => {
+    const result = await roomService.getRoomList(req);
+    return res.status(200).json({ result });
```

Comment:
```suggestion
    return res.status(200).json({ data:result });
```
key값 없이 보낼거면 json 안에 객체를 써서 보내는게 조금 의미가 없을 것 같습니다.